### DISCUSSION TITLE
What is the plan for premium support service for Elsa?

### DISCUSSION THREAD

**Discussion by kalpan123** [elsa-workflows/elsa-core#3638] [General]
Any information on premium support services for Elsa would be appreciated. Also, are there any plans for an enterprise version of Elsa?

**Comment by sfmskywalker** [comment]
Yes, me and my partners are in the process of setting up a company specialising in Elsa consultancy, support and enterprise services. Elsa itself will remain free and open source, but we are talking about offering enterprise-grade extensions.

We hope to be able to onboard premium support customers within the next 2 months. If you DM me your email address I can add you to our list so you'll be amongst the first to receive an update on this.

**Reply by dodyg** [reply]
I hope you consider regional pricing as well for the enterprise extensions. We are based in Egypt and we build a lot of software funded by international organizations for public software locally. Enterprise pricing for Europe/US simply out of budget for organizations outside these continents to due currency  and economic power differences.  

I understand the labour cost differences in terms support for uniform pricing but for extensions, regional pricing should be doable. This will allow you to capture revenues for a market that is usually underserved.

**Reply by sfmskywalker** [reply]
Thanks for raising this topic, we'll certainly keep this in mind.

**Comment by kalpan123** [comment]
Dear Sipke,


Thank you for the prompt response.

You can communicate to my official email ***@***.***


On Thu, 19 Jan 2023 at 17:38, Sipke Schoorstra ***@***.***>
wrote:

> Yes, me and my partners are in the process of setting up a company
> specialising in Elsa consultancy, support and enterprise services. Elsa
> itself will remain free and open source, but we are talking about offering
> enterprise-grade extensions.
>
> We hope to be able to onboard premium support customers within the next 2
> months. If you DM me your email address I can add you to our list so you'll
> be amongst the first to receive an update on this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/elsa-workflows/elsa-core/discussions/3638#discussioncomment-4727005>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIDUAGQT3G2RKRMWABHQGPLWTEVFJANCNFSM6AAAAAAUAGTKFM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Jay Shree Krishna

Thanks & Regards,
Kalpan K. Vaghela

(M)  +91 9408 555515

**Reply by kalpan123** [reply]
Hi, can you please update me on the same?

**Reply by sfmskywalker** [reply]
Hi @kalpan123 , your email address is redacted. Please send me a direct message at sipkeschoorstra@outlook.com and I'll add your email address to the list. Thanks.
